rd-cpp/thirdparty/clsocket/src/PassiveSocket.cpp

```diff
@@ -213,10 +209,9 @@ CActiveSocket *CPassiveSocket::Accept() {
     // Wait for incoming connection.
     //--------------------------------------------------------------------------
     if (pClientSocket != NULL) {
-        CSocketError socketErrno = SocketSuccess;
+        CSocketError socketErrno;
```

Comment:
Are you sure we don't want to initialize this variable?

Reply:
It is guaranteed to be initialised below either 225 or 239 line. So assignment here is noop.